installer/targets/ros-map_server/install.yaml

```diff
@@ -4,3 +4,9 @@
     url: https://github.com/tue-robotics/navigation.git
     sub-dir: map_server
     version: kinetic-devel
+
+- type: target
```

Comment:
We have to check what this does. It might just happen that this installs a lot of stuff on the robot pcs that we actually don't want there...

Reply:
So move to install.bash and surround with if robot!=real? The tue-save-map function can also be placed in an if robot!=real. So it is available on the robot. Good solution?

Reply:
@jlunenburg please read my previous comment